### PR TITLE
Change Requests

### PR DESCRIPTION
**Pull request by zkhin**
https://github.com/SeedCompany/cord-field/issues/788
https://github.com/SeedCompany/cord-field/issues/793
https://github.com/SeedCompany/cord-api-v3/issues/1788
API PR: https://github.com/SeedCompany/cord-api-v3/pull/1763

### UI Updates
image
image
image

### CODE REVIEW

**Comment by michaelmarshall** [comment]
component looks good to me. this work is tracking with the api side as it comes in.

**Comment by zkhin** [comment]
Updated the components regarding CRUD endpoints.

File: src/scenes/Projects/Update/UpdateProjectDialog.tsx [hunk 1]
```diff
@@ -183,6 +191,7 @@ export const UpdateProjectDialog = ({
             updateEngagementDateRanges(cache, project);
             updatePartnershipsDateRanges(cache, project);
           },
+          refetchQueries: [GQLOperations.Query.ProjectChangesOverview],
```
Comment (zkhin) [inline src/scenes/Projects/Update/UpdateProjectDialog.tsx:194]: Need to update cache directly.

**Review by CarsonF — APPROVED** [review]
🎉 🎉 🎉

**Pull request merged**